src/map/swipe.js

```diff
@@ -135,6 +135,13 @@ class SwipeController {
     this.resizeObserver_.observe(this.draggableElement_.parent().get(0));
   }
 
+  /**
+   * Allows you to deactivate the swiper on click of the close button.
+   */
+  deactivate() {
+    this.layer = null;
```

Comment:
```suggestion
    this.layer = null;
    this.map.render();
```
Otherwise the layer is now fully displayed after the deactivation.